R/use_r_workflows.R

```diff
@@ -224,7 +224,9 @@ use_create_cov_badge <- function(workflow_name = "call-create-cov-badge.yml", us
 #'  the workflow started.
 #' @param build_trigger Select the build trigger. Options are to run on pushing
 #'  commits to main ("push_to_main", the default); run when a pull request is
-#'  opened, reopened, or updated ("pull_request"); run manually with the
+#'  opened, reopened, or updated ("pull_request"); run when a comment containing
+#'  the command `\doc-and-style` is made on a pull request by people with write permissions
+#'  on the repository ("pr_comment"); run manually with the
 #'  workflow_dispatch trigger ("manually"); run on the default branch (usually
 #'  main) once a week ("weekly").
```

Comment:
There is no guidance regarding how to use multiple build triggers and the function doesn't allow for a user to specify that. For example, I wanted to add the ability to use the pr_comment to ss3sim and tried to use this function but it asked me if I wanted to overwrite the file that I already had named "call-doc-and-style-r.yml". What should users do if they want multiple build triggers?

Reply:
Right now, it is a work in progress! See https://github.com/nmfs-ost/ghactions4r/pull/220

In the meantime, users can manually add additional build triggers as desired after they have run ghactions4r::doc_and_style_r(). I added a sentence in the documentation to clarify.